src/core/middleware/requestId.js

```diff
@@ -1,17 +1,17 @@
 /* @flow */
 import httpContext from 'express-http-context';
 import { v4 as uuidv4 } from 'uuid';
-import type { $Request, $Response, NextFunction, Middleware } from 'express';
+import { $Request, $Response, NextFunction, Middleware } from 'express';
 
 import { AMO_REQUEST_ID_HEADER } from 'core/constants';
 
 // This middleware adds a correlation ID to the HTTP context and response.
 const requestId = (
-  req: $Request,
-  res: $Response,
-  next: NextFunction,
+  req: typeof $Request,
```

Comment:
mm is this new? I thought `$Request` was already a flow type.

Reply:
I'm not sure how it could be. In the existing code, we are importing `$Request` as a `type` from `'express'`, but `'express'` isn't covered by Flow, so it's the same problem as when we're trying to import a `type` from our own module that isn't covered by Flow. I also tried to find `$Request` (as well as `NextFunction` and `Middleware`) in the code for `express` and I couldn't find it anywhere. I might just not be checking in the right place though. Can you point me to the place where you believe these things are defined as types to Flow?

Reply:
mm I thought express had flow coverage in the past. Maybe they changed that? I also thought we were using flow-typed but nope. Maybe express was providing flow types in the past? Ha well..